src/widget/text_box.rs

```diff
@@ -340,6 +273,132 @@ impl<'a, F> TextBox<'a, F> {
 
 }
 
+fn get_clicked_elem<C, F>(text: &str, point: Point, args: &UpdateArgs<TextBox<F>, C>) -> Elem
+                                                                        where C: CharacterCache,
+                                                                        F: FnMut(&mut String) {
```

Comment:
This formatting seems unconventional, if you could try and follow the styling of the rest of conrod that would be preferred, i.e.
```rust
fn get_clicked_elem<C, F>(text: &str, point: Point, args: &UpdateArgs<TextBox<F>, C>) -> Elem
    where C: CharacterCache,
          F: FnMut(&mut String)
{
    ...
}
```